File: server/app/models/OutbreakDatapoint.js
var mongoose	    = require('mongoose');
var Schema			= mongoose.Schema;

var OutbreakDatapointSchema	= new Schema({
	_id: String,
	parent_id: String,
	date: Date,
	latitude: Number,
	longitude: Number,
	country: String,
	cases: Number,
	deaths: Number,
	unconfirmed: Number,
	notes: String
}, {collection: 'outbreak_datapoints'});

module.exports = mongoose.model('OutbreakDatapoint', OutbreakDatapointSchema);

File: ios-src/Ebola/OutbreakDatapoint.m
//
//  Case.m
//  Outbreak
//
//  Created by Peter on 11/4/14.
//  Copyright (c) 2014 Peter Kazazes. All rights reserved.
//

#import "OutbreakDatapoint.h"
#import "OutbreakDatapoint.h"


@implementation OutbreakDatapoint

@dynamic country;
@dynamic date;
@dynamic idString;
@dynamic latitude;
@dynamic longitude;
@dynamic notes;
@dynamic parentId;
@dynamic unconfirmed;
@dynamic cases;
@dynamic deaths;
@dynamic child;
@dynamic parent;

- (CLLocation *)location {
    return [[CLLocation alloc] initWithLatitude:[self.latitude doubleValue]  longitude:[self.longitude doubleValue]];
}

@end
